assets/sass/components/_badges-tags.scss

```diff
@@ -0,0 +1,12 @@
+a.govuk-link {
+  font-weight: 400;
```

Comment:
Perhaps I'm missing something, but does this change the weight of all of the links in the app? Is that the intention?

Reply:
Yes it was meant to be wrapped around the table tag we had a bit of a git mashup yesterday forgot to add in